src/Aspire.Hosting/ApplicationModel/ContainerFileSystemCallbackAnnotation.cs

```diff
@@ -84,12 +84,12 @@ public sealed class ContainerFileSystemCallbackAnnotation : IResourceAnnotation
     /// <summary>
     /// The UID of the default owner for files/directories to be created or updated in the container. Defaults to 0 for root.
     /// </summary>
-    public int DefaultOwner { get; init; }
+    public int? DefaultOwner { get; init; }
```

Comment:
Should we update the XML Docs about the "Defaults to 0 for root"?

Reply:
Yeah, updated to clarify the final UID/GID will default to 0 if the property is null.